frontend/packages/topology/src/components/quick-search/QuickSearchContent.tsx

```diff
@@ -17,6 +17,7 @@ interface QuickSearchContentProps {
   selectedItem: CatalogItem;
   onSelect: (itemId: string) => void;
   viewAll?: CatalogLinkData[];
+  closeModal?: () => void;
```

Comment:
Why is `closeModal` optional here and required in list and details component?
```suggestion
  closeModal: () => void;
```

Reply:
fixed